frontend/src/reducer/notificationPage.js

```diff
@@ -286,14 +286,40 @@ export default function notificationPage (state = defaultNotificationsObject, ac
     }
 
     case `${READ}/${NOTIFICATION}`: {
```

Comment:
Judging by the complexity of this action reducer, I would expect unit tests for it.